robottelo/ui/base.py

```diff
@@ -429,6 +429,9 @@ def auto_complete_search(self, go_to_page, entity_locator, partial_name,
         self.wait_for_ajax()
         strategy, value = common_locators['auto_search']
         self.click((strategy, value % name))
+        # Intentionally clicking second time because sometimes first time
+        # click just selects/highlights the item but not clicking it
+        self.click((strategy, value % name))
```

Comment:
@oshtaier , yeah true. It looks like this is not related to "the element is not into the visible area of
the browser window".